source/MRViewer/MRSurfaceManipulationWidget.h

```diff
@@ -94,6 +105,8 @@ class MRVIEWER_CLASS SurfaceManipulationWidget :
     VertScalars pointsShift_;
     VertScalars editingDistanceMap_;
     VertScalars visualizationDistanceMap_;
+    VertBitSet changedRegion_;
```

Comment:
is it needed only for `updateUV` function?

Reply:
yes (main idea is not to recalculate all uv coords, but only in changed vertices)